### DISCUSSION TITLE
How do you identify a container under another container?

### DISCUSSION THREAD

**Discussion by andreVik** [flet-dev/flet#978] [Q&A]
Hello everyone, I had a somewhat special request, is it possible to identify a container below in another container? For example imagine a blue container that is under a purple container is it possible with a function to understand if there is something under the purple container? And what more do you show me on the screen?

Thanks everyone

Update:

I attach a code example:
"""
def game(self: ft.Page):
self.bgcolor = ft.colors.WHITE
self.window_bgcolor = ft.colors.WHITE
self.window_center()
def movable(e):
while True:
for i in range(0,900,4):
b.top = i
sleep(0.01)
self.update()
else:
break
b = ft.Container(
width=100,
height=100,
alignment=ft.alignment.center,
bgcolor=ft.colors.BLUE_300,
border_radius=5,
animate = 1000,
)
f = ft.Container(
width=1000,
height=100,
bgcolor=ft.colors.PURPLE,
border_radius=5,
animate = 1000,
top=900,
)
a = ft.Stack(
controls=[
b,f
]
)
self.add(a)
sleep(2)
movable(None)
"""
In this case there is a box going down that then to end up in the purple container, I want it to be detected when the blue container goes down into the purple container and then tell me whether it is present or not with a print() for example.

**Comment by ghost** [comment]
do you mean something like this, a container inside a container?
```
import flet as ft
container_1 = ft.Container(content=ft.Container(content=ft.Text(value='some text')))
container_2 = container_1.content
```

or do you mean the next object in a column with "under"? e.g.

```
import flet as ft
my_column = ft.Column(controls= [ft.Container(content=ft.Text(value='container1')), ft.Container(content=ft.Text(value='container2')) ])

container_1 = my.column.controls[0]
container_2 = my column.controls[1]
```

**Comment by ndonkoHenri** [comment]
You could grab it from the `Control` which holds the both of them. 
If it's maybe a `Stack`, and you have two containers in its controls, then you could then grab the first item from the stack's `controls` list, which represents the control under.

**Comment by andree-2** [comment]
I attach a code example:
"""
def game(self: ft.Page):
    self.bgcolor = ft.colors.WHITE
    self.window_bgcolor = ft.colors.WHITE
    self.window_center()
    def movable(e):
            while True:
                for i in range(0,900,4):
                    b.top = i
                    sleep(0.01)
                    self.update()
                else:
                    break
    b = ft.Container(
                    width=100,
                    height=100,
                    alignment=ft.alignment.center,
                    bgcolor=ft.colors.BLUE_300,
                    border_radius=5,
                    animate = 1000,
                )
    f = ft.Container(
                    width=1000,
                    height=100,
                    bgcolor=ft.colors.PURPLE,
                    border_radius=5,
                    animate = 1000,
                    top=900,
                )
    a = ft.Stack(
            controls=[
                b,f
            ]
    )
    self.add(a)
    sleep(2)
    movable(None)
"""
In this case there is a box going down that then to end up in the purple container, I want it to be detected when the blue container goes down into the purple container and then tell me whether it is present or not with a print() for example.
